service/selectors/entities/channels.js

```diff
@@ -148,3 +149,22 @@ export const getAutocompleteChannelWithSections = createSelector(
         return channels;
     }
 );
+
+export const canManageChannelMembers = createSelector(
+    getCurrentChannel,
+    getCurrentChannelMembership,
+    getCurrentTeamMembership,
+    getUsers,
+    getCurrentUserId,
+    (channel, channelMembership, teamMembership, allUsers, currentUserId) => {
+        const user = allUsers[currentUserId];
+        const roles = `${channelMembership.roles} ${teamMembership.roles} ${user.roles}`;
```

Comment:
@thomchop reverted the `Ready to merge` label and added awaiting submitter action to fix this as @jarredwitt mentioned before merging